### ISSUE TITLE
forkSense output all 0s for EdU/BrdU

### ISSUE THREAD

**Issue by ddubocan** [MBoemo/DNAscent#34]
Hi,

After reading the docs I was under the impression that the output of forkSense should be a file where EdU/BrdU regions are encoded with a boolean (0 or 1). 
I am noticing the opposite behavior -- the output of forkSense is all 0s in both columns, and I lose >95% of the reads that are present in the DNAscent Detect output.

This is an example header:

```#DetectFile detect_dnascent
#Threads 24
#Compute CPU
#SystemStartTime 24/08/2023 14:34:25
#Software ~/DNAscent/
#Version 3.1.2
#Commit 3542c4b2f5bf35afd6a5961e14a1ce4c8a7ea692
#EstimatedRegionBrdU 0.700077
#EstimatedRegionEdU 0.040431
>55f8fae2-4f62-41db-a2dc-1bd946d463bd chr10 6958586 7011068 fwd
6958586 0       0
6958590 0       0
6958592 0       0
6958594 0       0
6958596 0       0
```

Visual inspection shows the 2nd and 3rd columns are exclusively populated with 0s. 
Also running ```grep -v ">" forkSense_output | grep -v "#" | datamash sum 2,3 ```
yields 0 0.

tldr; Only reads with no incorporated regions are output in forkSense. 
How can I figure out what's going wrong?

-Danilo

**Comment by ddubocan** [comment]
I've reproduced this on a couple of files, including those which seem to have solid levels of EdU,BrdU incorporation. 
Additionally, when visually looking at individual reads there are clearly patches of EdU,BrdU incorporation yet those reads are emitted from the forkSense output.

**Comment by ddubocan** [comment]
Thanks for the quick response!
I see this behavior in files where the incorporation of both analogues is higher.
The pulse order we EdU,BrdU.

The issue is I am not seeing the behavior described in your second point. Every read contains exclusively 0s in both columns. Even if there is not a fork there, shouldn't we expect some encoding of being an analogue positive region?

![Screen Shot 2023-08-24 at 3 16 09 PM](https://github.com/MBoemo/DNAscent/assets/55263857/e8d31e66-c4e2-4c87-ac44-504c484c282a)
